Headers/AppKit/NSUserInterfaceItemIdentification.h

```diff
@@ -1,44 +1,56 @@
-/** NSUserInterfaceItemIdentification.h
-   <abstract>Associate a unique identifier with objects in your user interface</abstract>
-
-   Copyright <copy>(C) 2017 Free Software Foundation, Inc.</copy>
-
-   Author: Daniel Ferreira <dtf@stanford.edu>
```

Comment:
Could you please restore Daniel's dates. He created the original form of this file and should be mentioned.

Reply:
Yes, I obliterated the file by accident, in fact... will do.